src/source/Ice/NatBehaviorDiscovery.c

```diff
@@ -0,0 +1,404 @@
+/**
+ * Kinesis Video Producer Ice Utils
+ */
+#define LOG_CLASS "NatTypeDiscovery"
+#include "../Include_i.h"
+
+#define DEFAULT_NAT_TEST_MAX_BINDING_REQUEST_COUNT                  5
+#define DEFAULT_TEST_NAT_TEST_RESPONSE_WAIT_TIME                    500 * HUNDREDS_OF_NANOS_IN_A_MILLISECOND
+
+#define NAT_BEHAVIOR_NONE_STR                                       (PCHAR) "NONE"
```

Comment:
Undefined?

Reply:
I think NONE is what our code base is using.